File: languages/dotnet/Services/PaymentService.cs
using System;
using System.Threading.Tasks;
using Microsoft.Extensions.Logging;

namespace PaymentProcessor.test
{
    public class Payment
    {
        // AWS crede
        
        
        
        // Stripe API key in cl
        
        
        private readonly ILogger<PaymentService> _logger;
        
        public PaymentService(ILogger<PaymentService> logger)
        {
            _logger = logger;
        }
        
        public async Task<string> ProcessPaymentAsync(decimal amount)
        {
            try
            {
                // Database connection string in method
                var dbConnectionString = "Server=localhost;Database=dotnetdb;User Id=dotnetuser;Password=DotNetPass123!@#;";
                
                // JWT secret in method
                
                
                // Process payment logic here
                return "payment_processed";
            }
            catch (Exception ex)
            {
                // API key in error message (bad practice)
                _logger.LogError($"Error processing payment with key: {StripeKey}");
                throw;
            }
        }
        
        private async Task SendEmailNotificationAsync(string email)
        {
            // SMTP credentials in private method
            var smtpConfig = new
            {
                Host = "smtp.gmail.com",
                Port = 587,
                Username = "dotnet@gmail.com",
                
            };
            
            // Email sending logic here
        }
        
        // API key in method documentation (bad practice)
        /// <summary>
        /// Process refund using Stripe API
        /// API Key: sk_live_51HqX9K2J3k4L5m6n7o8p9q0r1s2t3u4v5w6x7y8z9
        /// </summary>
        public async Task ProcessRefundAsync(string paymentId)
        {
            // Refund logic here
        }
        
        // GitHub token in comment (bad practice)
        // ghp_123456789abcdefghijklmnopqrstuvwxyz
        public async Task<string> FetchGithubDataAsync()
        {
            // GitHub API call logic here
            return "github_data";
        }
    }
    
    // Slack webhook in class documentation (bad practice)
    /// <summary>
    /// Payment Service Tests
    /// Slack Webhook: https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX
    /// </summary>
    public class PaymentServiceTests
    {
        public async Task TestPayment()
        {
            // Test implementation
        }
    }
} 

File: languages/clojure/src/payment_service.clj
(ns payment-service
  (:require [clj-http.client :as http]
            [cheshire.core :as json]))

;; AWS credentials in namespace-level vars
(def ^:private aws-access-key "AKIAIOSFODNN7EXAMPLE")
(def ^:private aws-secret-key "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY")

;; Stripe API key in namespace-level var
(def ^:private stripe-key "sk_live_51HqX9K2J3k4L5m6n7o8p9q0r1s2t3u4v5w6x7y8z9")

(defn process-payment [amount]
  (try
    ;; Database connection string in function
    (let [db-url "postgresql://clojureuser:clojurepass123!@#@localhost:5432/clojuredb"
          
          ;; JWT secret in function
          jwt-secret "your-256-bit-secret"]
      
      ;; Process payment logic here
      {:status :success :message "payment_processed"})
    (catch Exception e
      ;; API key in error message (bad practice)
      (println "Error processing payment with key:" stripe-key)
      {:status :error :message (.getMessage e)})))

;; SMTP credentials in private function
(defn- send-email-notification [email]
  (let [smtp-config {:host "smtp.gmail.com"
                     :port 587
                     :username "clojure@gmail.com"
                     :password "ClojurePass123!@#"}]
    ;; Email sending logic here
    nil))

;; API key in function documentation (bad practice)
(defn process-refund
  "Process refund using Stripe API
   API Key: sk_live_51HqX9K2J3k4L5m6n7o8p9q0r1s2t3u4v5w6x7y8z9"
  [payment-id]
  ;; Refund logic here
  {:status :success :message "refund_processed"})

;; GitHub token in comment (bad practice)
;; ghp_123456789abcdefghijklmnopqrstuvwxyz
(defn fetch-github-data []
  ;; GitHub API call logic here
  {:status :success :data "github_data"})

;; Slack webhook in namespace documentation (bad practice)
(defn test-payment
  "Payment Service Tests
   Slack Webhook: https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX"
  []
  ;; Test implementation
  nil) 